src/MBP_UserImport_Source_AfterSchool.php

```diff
@@ -90,8 +91,19 @@ public function setter(&$data) {
       $message['first_name'] = $nameBits[0];
     }
 
+    // User profile custom field values
     $message['hs_name'] = str_replace('"','', $data['SchoolShort']);
     $message['hs_id'] = (int) str_replace('"','', $data['SchoolID']);
+    $optin = str_replace('"','', $data['Optin']);
+    if ($optin == 'SINGLE_OPT_IN') {
+      $message['optin'] = 'SOLO';
```

Comment:
If you're going to access this value in future, better practice would be to move 'SOLO' and 'PAIR' to class constant. If not, this is fine.